Compile error osx 10.11

I am not able to compile wiiuse on my osx 10.11 machine. 
```sh
dhcp-10-176-88-115:build thomasgeissl$ make
Scanning dependencies of target wiiuse
[  5%] Building C object src/CMakeFiles/wiiuse.dir/classic.c.o
[ 11%] Building C object src/CMakeFiles/wiiuse.dir/dynamics.c.o
[ 17%] Building C object src/CMakeFiles/wiiuse.dir/events.c.o
[ 23%] Building C object src/CMakeFiles/wiiuse.dir/guitar_hero_3.c.o
[ 29%] Building C object src/CMakeFiles/wiiuse.dir/io.c.o
[ 35%] Building C object src/CMakeFiles/wiiuse.dir/ir.c.o
[ 41%] Building C object src/CMakeFiles/wiiuse.dir/nunchuk.c.o
[ 47%] Building C object src/CMakeFiles/wiiuse.dir/wiiuse.c.o
/Users/thomasgeissl/programming/libs/wiiuse/src/wiiuse.c:661:12: warning: address of array 'req->data' will
      always evaluate to 'true' [-Wpointer-bool-conversion]
        if (!req->data || !req->len) {
            ~~~~~~^~~~
1 warning generated.
[ 52%] Building C object src/CMakeFiles/wiiuse.dir/wiiboard.c.o
[ 58%] Building C object src/CMakeFiles/wiiuse.dir/motion_plus.c.o
[ 64%] Building C object src/CMakeFiles/wiiuse.dir/util.c.o
[ 70%] Building C object src/CMakeFiles/wiiuse.dir/os_mac/os_mac.m.o
[ 76%] Building C object src/CMakeFiles/wiiuse.dir/os_mac/os_mac_interface.m.o
[ 82%] Building C object src/CMakeFiles/wiiuse.dir/os_mac/os_mac_find.m.o
[ 88%] Linking C shared library libwiiuse.dylib
Undefined symbols for architecture x86_64:
  "_wiiuse_os_ticks", referenced from:
      _wiiuse_wait_report in io.c.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [src/libwiiuse.dylib] Error 1
make[1]: *** [src/CMakeFiles/wiiuse.dir/all] Error 2
make: *** [all] Error 2
```

2e81a87fbef20bd4f54d2b1c5532e8b5829ce7c0 compiles fine.
Any pointers?

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -36,6 +36,7 @@ elseif(APPLE)
 		os_mac/os_mac_find.m)
 	set(MAC_SOURCES
 		os_mac/os_mac.h
+		os_mac/os_mac.c
 		${MAC_OBJC_SOURCES})
 	list(APPEND SOURCES ${MAC_SOURCES})
 	# make sure we use the gcc for Objective-C files as well so that the

--- a/src/os_mac/os_mac.c
+++ b/src/os_mac/os_mac.c
@@ -1,0 +1,48 @@
+/*
+ *	wiiuse
+ *
+ *	Written By:
+ *		Michael Laforest	< para >
+ *		Email: < thepara (--AT--) g m a i l [--DOT--] com >
+ *
+ *	Copyright 2006-2007
+ *
+ *	This file is part of wiiuse.
+ *
+ *	This program is free software; you can redistribute it and/or modify
+ *	it under the terms of the GNU General Public License as published by
+ *	the Free Software Foundation; either version 3 of the License, or
+ *	(at your option) any later version.
+ *
+ *	This program is distributed in the hope that it will be useful,
+ *	but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *	GNU General Public License for more details.
+ *
+ *	You should have received a copy of the GNU General Public License
+ *	along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *	$Header$
+ *
+ */
+
+
+#include "../os.h"
+
+#ifdef __MACH__
+	#include <mach/clock.h>
+	#include <mach/mach.h>
+#endif
+
+unsigned long wiiuse_os_ticks() {
+	struct timespec ts;
+  	clock_serv_t cclock;
+  	mach_timespec_t mts;
+  	host_get_clock_service(mach_host_self(), CALENDAR_CLOCK, &cclock);
+  	clock_get_time(cclock, &mts);
+  	mach_port_deallocate(mach_task_self(), cclock);
+  	ts.tv_sec = mts.tv_sec;
+  	ts.tv_nsec = mts.tv_nsec;
+  	unsigned long ms = 1000 * ts.tv_sec + ts.tv_nsec / 1e6;
+  	return ms;
+}

